### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Arpita-25**
Added a resource for web scraping. Might help!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# web-scraping
\ No newline at end of file
+# web-scraping
+
+Useful resource for web scraping: 
+https://www.udemy.com/share/101Wm63@b-HpjeXawuJPUQ68u2eMIDM80WEQFPXsoz_Xsrx4bARMLnsO2Gy5VYW1uyLBE-9d4g==/
```
Comment (Lakshita2002) [inline README.md:4]: Could you please add a few more resources to the README, thanks!

Comment (Arpita-25) [reply README.md:4]: Sure!

**Comment by Arpita-25** [comment]
Hey check it out! I added a complete crash course link on web scraping.

File: README.md (hunk 1 above)
Comment (Lakshita2002) [reply README.md:4]: That works, merging the PR!

**Pull request merged**